composer.json

```diff
@@ -32,9 +32,10 @@
         "issues": "http://github.com/heyday/silverstripe-elastica/issues"
     },
     "require": {
-        "php": ">=5.6.0",
-        "ruflin/elastica": "~5.3",
-        "symbiote/silverstripe-queuedjobs": "~4.0"
+        "php": ">=7.2",
+        "ruflin/elastica": "^7",
+        "elasticsearch/elasticsearch": "~7.3.0",
+        "symbiote/silverstripe-queuedjobs": "^4"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7"
```

Comment:
Are we able to use phpunit 9 yet in Silvwerstripe projects? I saw some work on it a few weeks ago

Reply:
Looks like it's restricted to Silverstripe 4.10, which isn't stable yet.